src/components/Pokemon.js

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+function Pokemon({name}) {
+    const [pokemon, setPokemon] = useState([]);
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`);
+                setPokemon(response.data);
+                // console.log(pokemon);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchData();
+    }, []);
+
+    return (
+        <>
+            {Object.keys(pokemon).length > 0 &&
```

Comment:
Deze check kan dan wat simpeler
```suggestion
            {pokemon &&
```